auth-web/src/components/auth/common/AccountLoginOptionPicker.vue

```diff
@@ -152,7 +151,7 @@ export default class AccountLoginOptionPicker extends Mixins(AccountChangeMixin,
   }
 
   private get showLearnMore (): boolean {
-    return LaunchDarklyService.getFlag(LDFlags.AuthLearnMore) || false
+    return true
```

Comment:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/cf0ceb8f-d074-4351-8a84-33f6bb4bd068)

Reply:
Same as above. Just remove v-if from line 32 (and this getter).

Reply:
Thanks. Looks good.